src/files/coverageparser.ts

```diff
@@ -213,17 +216,18 @@ export class CoverageParser {
                     return resolve(sections);
                 });
             } catch (error) {
-                this.handleError("lcov-parse", error);
+                this.handleError("lcov-parse", error, filename);
                 return resolve(new Map<string, Section>());
             }
         });
     }
 
-    private handleError(system: string, error: Error) {
+    private handleError(system: string, error: Error, filename?: string) {
```

Comment:
Added the file in question to the method signature, really this is just for error reporting so the user can track down the messed up files.

Reply:
As mentioned above, if we go the prepend to the error message near to the error logic we can revert these changes to the `handleError`.